docker-compose.dev.yml

```diff
@@ -43,7 +43,7 @@ services:
 
   ipfs:
     container_name: ipfs-vsc_dev
-    image: ipfs/kubo:v0.18.1
+    image: ipfs/kubo:v0.28.0
```

Comment:
Last Kubo version that does not break anything was v0.27.0 as certain `ipfs.object` APIs were removed in v0.28.0. Manual fs repo migrations will be required.